tmpl/blog/2016-summer-hackathon-roundup.md

```diff
@@ -46,17 +41,19 @@ Meanwhile, Qi Li worked on testing and adapting [simple-nat](https://github.com/
 * [Simple-NAT ethernet branch](https://github.com/yomimono/simple-nat/tree/ethernet-level-no-irmin)
 * [Mirage NAT with optional Irmin branch](https://github.com/yomimono/mirage-nat/tree/depopt_irmin)
 
-### MirageOS 3.0 API changes
+[Hannes Mehnert](https://www.cl.cam.ac.uk/~hm519/) recently published a purely functional [ARP package](https://github.com/hannesm/arp) and continued refining it (with code coverage via [bisect](http://bisect.x9c.fr)) during the hackathon.
```

Comment:
pls link to https://github.com/aantron/bisect_ppx (and in arp there's simply a `-pkg bisect_ppx` passed to ocamlbuild)